.github/workflows/create_release_assets.yml

```diff
@@ -51,6 +56,22 @@ jobs:
         if: ${{ matrix.platform != 'windows-latest' }}
         shell: bash
 
+      - name: Build Debian-based system binary and create package
+        run: |
+          rm -rf target/release
+          cargo build --release --features default
```

Comment:
The `default` feature will be enabled by default

Reply:
Fair enough.  I tend to favor being explicit about what's going on vs. being concise, which is why it's there, but I can certainly take out the `--features` option.